### DISCUSSION TITLE
Poll message through api

### DISCUSSION THREAD

**Discussion by mycodemm** [dimaskiddo/go-whatsapp-multidevice-rest#56] [Ideas]
can you add poll message feature in api?

**Comment by dimaskiddo** [comment]
@mycodemm poll message mean you want to store the messages first into the database and query it, or just like query it directly? some of it may have some pros and cons

**Comment by ibnux** [comment]
@dimaskiddo maybe vote message, polling system
https://faq.whatsapp.com/796470361614974/?cms_platform=web

but will be good if it can add webhook, everytime it received message, it send POST to url where WEBHOOK_URL set.
with parameters
**from**: number who send it
**to**: phone number who received it
**message**: the text message
**attachment**: if any and can

so user can do anything with the message, send it to Bot API, save it to database, or do something with Smarthome

**Comment by mycodemm** [comment]
> @mycodemm poll message mean you want to store the messages first into the database and query it, or just like query it directly? some of it may have some pros and cons

Better to store and then message, it will help to record votes

**Comment by dimaskiddo** [comment]
owh i see pak @ibnux @mycodemm it's poll votes, currently i'am on working to add this feature to make it more easy to use

**Reply by dimaskiddo** [reply]
Send Poll added in version [v1.7.1](https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/releases/tag/v1.7.1)
Cheers 🥂
